### PR TITLE
added rating component

### PR DESCRIPTION
**Pull request by rohanmathur91**
## Description:
 - Added rating component
 - Did few style fix
 
## Screenshot:
image

### CODE REVIEW

**Review by ishitakeshawani — APPROVED** [review]
LGTM

**Review by Aditi0505 — APPROVED** [review]
Looks good to me.

File: utilities/reset.css [hunk 1]
```diff
@@ -1,5 +1,5 @@
-@import url("https://fonts.googleapis.com/css2?family=Inter:wght@400;700;900&display=swap");
-@import url("https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css");
+@import "https://fonts.googleapis.com/css2?family=Inter:wght@400;700;900&display=swap";
+@import "https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css";
 
 /* reset */
```
Comment (ckahub) [inline utilities/reset.css:4]: remove line if not needed

**Review by ckahub — COMMENTED** [review]
looks good to me

**Pull request merged**